core/themes/basis/css/skin.css

```diff
@@ -169,7 +169,7 @@ fieldset .fieldset-legend {
 
 .header-menu a:hover,
 .l-header a:active {
-  background: #114a75;
+  text-decoration: underline;
```

Comment:
The `:active` color falls back to `#fffffe`:

<img width="471" alt="Screen Shot 2019-08-15 at 10 40 12 pm" src="https://user-images.githubusercontent.com/2423362/63121671-b1845d00-bfad-11e9-91e0-deb61f3e336e.png">

```suggestion
  text-decoration: underline;
  background: unset;
```